tensorflow/core/common_runtime/sycl/sycl_device_factory.cc

```diff
@@ -18,24 +18,34 @@ limitations under the License.
 #include "tensorflow/core/common_runtime/device_factory.h"
 #include "tensorflow/core/common_runtime/sycl/sycl_device.h"
 
+#include "tensorflow/core/common_runtime/sycl/sycl_util.h"
+
 namespace tensorflow {
 
 class SYCLDeviceFactory : public DeviceFactory {
  public:
   Status CreateDevices(const SessionOptions &options, const string &name_prefix,
                        std::vector<Device *> *devices) override {
-    int n = 1;
+
+    auto syclInterface = GSYCLInterface::instance();
+
+    size_t n = 1;
     auto iter = options.config.device_count().find("SYCL");
     if (iter != options.config.device_count().end()) {
       n = iter->second;
     }
+
     for (int i = 0; i < n; i++) {
-      string name = strings::StrCat(name_prefix, "/device:SYCL:", i);
+      string name = strings::StrCat(name_prefix, "/sycl:", i);
```

Comment:
Can yo revert this change ? All devices should be named /device:DEVICE_NAME:DEVICE_ID (with the exception of CPUs and GPUs, which can also be referred to as /cpu:cpu_id and /gpu:gpu_id in order to preserve backward compatibility).